support/build/config-submenu.sh

```diff
@@ -79,25 +79,25 @@ if [ ! -z "$ARG_OUT" ]; then
 	fi
 	[ -f "$ARG_OUT" ] && rm -f "$ARG_OUT"
 
-	# open fd:7
-	exec 7<>"$ARG_OUT"
+	# create and open a new file descriptor
+	exec {fd}<>"$ARG_OUT"
```

Comment:
Can we use capital letters for the `FD`, this would be inline with the rest of the style in this file. Otherwise, this is a great fix!